### PR TITLE
New package: hashboot-0.9.6

### PR DESCRIPTION
**Pull request by teldra**

### CODE REVIEW

File: srcpkgs/hashboot/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+short_desc="Check integrity of files in /boot, mbr and bios"
+maintainer="teldra <teldra@rotce.de>"
+license="Hugware"
+homepage="https://github.com/tastytea/hashboot"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=f45805430d77ed935616217bd78dbc79b8ee9e1ff1ffe6536fe1664a7243ae05
+
+do_install() {
+	tar xf ${version}.tar.gz
+	cd ${pkgname}-${version}
```
Comment (Johnnynator) [inline srcpkgs/hashboot/template]: This two lines should be not needed when you don't set a `build_style` at all.

Comment (teldra) [reply srcpkgs/hashboot/template]: I tried but sadly it does not work.

File: srcpkgs/hashboot/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'hashboot'
+pkgname=hashboot
+version=0.9.6
+revision=1
+build_style=fetch
```
Comment (maxice8) [inline srcpkgs/hashboot/template]: remove

File: srcpkgs/hashboot/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+short_desc="Check integrity of files in /boot, mbr and bios"
+maintainer="teldra <teldra@rotce.de>"
+license="Hugware"
+homepage="https://github.com/tastytea/hashboot"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=f45805430d77ed935616217bd78dbc79b8ee9e1ff1ffe6536fe1664a7243ae05
+
+do_install() {
+	tar xf ${version}.tar.gz
```
Comment (maxice8) [inline srcpkgs/hashboot/template]: these 2 lines as well

File: srcpkgs/hashboot/template [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+	cd ${pkgname}-${version}
+	vinstall kernel-hook 755 etc/kernel.d/post-install 99-hashboot
+	vinstall kernel-hook 755 etc/kernel.d/post-remove 99-hashboot
+	vinstall initscript.runit 644 etc/runit/core-services 04-hashboot.sh
+	vbin hashboot
+	vlicense LICENSE Hugware
```
Comment (maxice8) [inline srcpkgs/hashboot/template]: just leave at LICENSE

File: srcpkgs/hashboot/template [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+revision=1
+short_desc="Check integrity of files in /boot, mbr and bios"
+maintainer="teldra <teldra@rotce.de>"
+license="Hugware"
+homepage="https://github.com/tastytea/hashboot"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=f45805430d77ed935616217bd78dbc79b8ee9e1ff1ffe6536fe1664a7243ae05
+
+do_install() {
+
```
Comment (maxice8) [inline srcpkgs/hashboot/template]: extra empty line

**Comment by teldra** [comment]
Problems should be solved.

**Pull request merged**